examples/agents_tracing-eval_mlflow/nps_agent/2_deploy/npsagent.py

```
@@ -0,0 +1,72 @@
+        result = await Runner.run(agent, prompt)
+        return result.final_output
+
+
+# ---------------------------------------------------------------------------
+# MLflow ResponsesAgent — wraps run_nps_agent into an HTTP API for deployment
+# ---------------------------------------------------------------------------
+class NPSResponsesAgent(ResponsesAgent):
+    def predict(self, request: ResponsesAgentRequest) -> ResponsesAgentResponse:
+        try:
+            result = asyncio.run(run_nps_agent(request.input))
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)